airflow/jobs/scheduler_job.py

```diff
@@ -691,9 +692,7 @@ def _process_task_instances(self, dag, task_instances_list, session=None):
         for run in active_dag_runs:
             self.log.debug("Examining active DAG run: %s", run)
             # this needs a fresh session sometimes tis get detached
```

Comment:
This comment doesn't really make any sense anymore does it? It doesn't seem to relate to the code at least.